### DISCUSSION TITLE
Allow customisation of createOptionForm on Select controls

### DISCUSSION THREAD

**Discussion by RicLeP** [filamentphp/filament#11672] [Ideas]
The default select `createOptionForm` button on `Select` isn’t very visible. It would be great to be able to customise the appearance. At the minimum would be colour and label / icon. Maybe even being able to set the position - end of control, button above control, below the control etc.

<img width="392" alt="image" src="https://github.com/filamentphp/filament/assets/19726976/ec0078f3-382a-4c04-9c82-070a9fd11e7f">

**Comment by dissto** [comment]
I think you can. It's just a "regular" Action which allows the same styling options as any other, you just need to inject it to modify.

```php
->createOptionAction(function (Forms\Components\Actions\Action $action) {
    $action
        ->button()
        ->color('danger');
})
```
